al_servicec.js

```diff
@@ -164,19 +173,33 @@ class IngestC extends AlServiceC {
         return this.post(`/data/secmsgs`, payload);
     }
 
-    sendVpcFlow(data) {
+    sendVpcFlow(data, invokedBy) {
         let payload = {
             json : false,
             headers : {
                 'Content-Type': 'alertlogic.com/cwl-json',
-                'x-invoked-by' : 'lambda_function',
+                'x-invoked-by' : this._functionType,
                 'Content-Encoding' : 'deflate',
                 'Content-Length' : Buffer.byteLength(data)
             },
             body : data
         };
         return this.post(`/data/vpcflow`, payload);
     }
+    
+    sendAicspmsgs(data, invokedBy) {
```

Comment:
Just out of curiosity what does it stand for `Aicsp` ? Maybe more camel case here ;-)?

Reply:
it stands for "appliance in cloud send parsed" -  I don't think we need more camels here =)